.eslintrc.js

```diff
@@ -161,6 +161,8 @@ module.exports = {
             // Always try to resolve types under `<root>/@types` directory even
             // it doesn't contain any source code, like `@types/unist`
             alwaysTryTypes: true,
+            // Allow importing .cjs files
+            extensions: ['.js', '.cjs'],
```

Comment:
Let me know if this is something you’d rather not change in your ESLint config, if that’s the case I can publish a new version of the package with `.cjs.js` for the CJS export.